src/erlfmt.erl

```diff
@@ -272,7 +274,81 @@ format_file_range(FileName, StartLocation, EndLocation, Options) ->
 ) ->
     {ok, string(), [error_info()]}
     | {error, error_info()}.
-format_string_range(String, StartLocation, EndLocation, Options) ->
+format_string_range(Original, StartLocation, EndLocation, Options) ->
+    FileName = "nofile",
+    Pragma = proplists:get_value(pragma, Options, ignore),
+    {ok, Nodes, Warnings} = read_nodes_string(FileName, Original, Pragma),
```

Comment:
Wouldn't setting the `nofile` here result in bad diagnostics when called from the `format_file_range` function?

Reply:
Oups, good point, thanks!